packages/flutter/test/rendering/layers_test.dart

```diff
@@ -723,15 +723,15 @@ void main() {
   test('OpacityLayer dispose its engineLayer if there are no children', () {
     final OpacityLayer layer = OpacityLayer(alpha: 128);
     final FakeSceneBuilder builder = FakeSceneBuilder();
-    layer.addToScene(builder);
+    layer.addToSceneWrapped(builder);
     expect(layer.engineLayer, null);
 
     layer.append(PictureLayer(Rect.largest)..picture = FakePicture());
-    layer.addToScene(builder);
+    layer.addToSceneWrapped(builder);
     expect(layer.engineLayer, isA<FakeOpacityEngineLayer>());
 
     layer.removeAllChildren();
-    layer.addToScene(builder);
+    layer.addToSceneWrapped(builder);
     expect(layer.engineLayer, null);
   });
 
```

Comment:
Test case to test the newly added assert is missing.

Reply:
done (waiting ci)